.github/workflows/cloudformation-ci.yml

```diff
@@ -99,6 +99,8 @@ jobs:
       - name: Wait for agents to enroll
         id: wait-for-agents
         working-directory: ${{ env.WORKING_DIR }}/fleet_api
+        env:
+          TIMEOUT: 20
```

Comment:
can you please comment what time unit it is and where it is being used (it can be mistaken as the workflow timeout)